spec/requests/providers/means/remove_state_benefits_controller_spec.rb

```diff
@@ -0,0 +1,110 @@
+require "rails_helper"
+
+RSpec.describe Providers::Means::RemoveStateBenefitsController do
+  let(:legal_aid_application) { create(:legal_aid_application, :with_applicant) }
+  let(:login) { login_as legal_aid_application.provider }
+  let(:state_benefit_transaction_type) { create(:transaction_type, :benefits) }
+  let(:chosen_transaction) do
```

Comment:
nit linebreaks between multiline blocks - i guess rubocop has this cop disabled

Reply:
I think it only expects line breaks between changes, e.g. between subject and let, and let and before